src/celengine/galaxy.cpp

```diff
@@ -161,15 +161,18 @@ float Galaxy::getBrightnessCorrection(const Eigen::Vector3f &offset) const
 
     // corrections to avoid excessive brightening if viewed e.g. edge-on
     float brightness_corr = 1.0f;
-    if (type < GalaxyType::E0 || type > GalaxyType::E3) // all galaxies, except ~round elliptics
+    if (type != GalaxyType::Irr)
```

Comment:
```suggestion
    if (type != GalaxyType::Irr && (type < GalaxyType::E0 || type > GalaxyType::E3))  // all galaxies, except ~round elliptics and irregular
```
and the rest of original code remains the same